src/event.js

```diff
@@ -488,9 +488,10 @@ jQuery.event = {
 		// Find delegate handlers
 		// Black-hole SVG <use> instance trees (#13180)
 		//
-		// Support: Firefox
-		// Avoid non-left-click bubbling in Firefox (#3861)
-		if ( delegateCount && cur.nodeType && ( !event.button || event.type !== "click" ) ) {
+		// Support: Firefox<=42+
+		// Avoid non-left-click in FF but don't block IE radio events (#3861, gh-2343)
+		if ( delegateCount && cur.nodeType &&
+			( event.type !== "click" || isNaN( event.button ) || event.button < 1 ) ) {
```

Comment:
That's true both ways, and there's no practical performance difference. It's just smaller without the `isNaN`, and the explanation is already handled pretty well by our support comment (which could become "Suppress clicks of non-primary mouse buttons" if you want to increase clarity a little more).